lib/resolver-factory.ts

```diff
@@ -90,6 +91,23 @@ export function createResolver(definition: Definition) {
         return result;
       }) as any;
     }
+
+    @Mutation(() => SuccessResponse)
+    async [`remove${definition.name}`](
+      @Args('id', { type: () => graphql.GraphQLID }) id: string,
+    ) {
+      try {
+        const result = await this.model.findByIdAndRemove(id);
+
+        if (result) {
+          return { success: true };
+        } else {
+          throw new Error(`No ${definition.name} found with ID: ${id}`);
+        }
+      } catch (error) {
+        throw new Error(`Failed to remove ${definition.name}`);
+      }
```

Comment:
Just some code styling.
* Early exit to reduce indents
* No need additional try catch, unexpected errors will be caught in other places (with Nest) 
* Should throw GraphqlError instead of normal Error (Sorry I threw normal errors, could you fix that as well?)

```suggestion
        const removed = await this.model.findByIdAndRemove(id);
        if (util.isNil(removed)) throw new GraphqlError('sth');
        return { success: true };
```

Reply:
also to cover throw case, can you call remove with non-exists id like `000000000000000000000000`